gm4_combat_expanded/data/gm4_combat_expanded/predicates/mob/list_air_spawn.json

```diff
@@ -0,0 +1,68 @@
+{
+  "condition": "minecraft:alternative",
+  "terms": [
+    {
+      "condition": "minecraft:entity_properties",
+      "entity": "this",
+      "predicate": {
+        "type": "minecraft:blaze"
```

Comment:
While true, blazes and ghasts fly right after spawning, and slimes/magma cubes often jump. Because of this they are almost always in the air when the clock runs and processes them, so I'm considering them 'air spawns'.

The only mob this should have a noticeable effect on is blazes, which will be modified even when from a spawner, but I think since that's where blazes are most often fought anyway this is a good thing. Blazes also specifically do not get extra health as to not break blaze farms